taxi/forms.py

```diff
@@ -0,0 +1,48 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.contrib.auth import get_user_model
+import re
+from .models import Car, Driver
+
+User = get_user_model()
+
+
+class DriverCreationForm(forms.ModelForm):
```

Comment:
According to the checklist, you should use `UserCreationForm` while creating a user. Currently, `DriverCreationForm` is inheriting from `forms.ModelForm`, which does not meet the requirement.